Dockerfile

```
@@ -1,32 +1,67 @@
 # DockerFile for grpc4bmi. Installs the C++ bindings in the /usr/local prefix directory in the container. If you are
 # planning to run a container with your BMI-enabled model and communicate with it using grpc4bmi, you can use this as a
 # base image for your model
-FROM ubuntu:bionic
-MAINTAINER Gijs van den Oord <g.vandenoord@esciencecenter.nl>
-RUN apt-get update
+FROM ubuntu:24.04
+LABEL author="Gijs van den Oord"
+LABEL email="g.vandenoord@esciencecenter.nl"
+
```

We can refer to this repo here;
```suggestion
LABEL org.opencontainers.image.source="https://github.com/eWaterCycle/grpc4bmi"

```
if this image is hosted on the GH container registry it will automatically show up in this repository under "packages".